src/opt/DebugPass.cpp

```diff
@@ -58,18 +52,9 @@ void DebugPassFile::run(llvm::Module &module, bool init) {
     module.print(OS, 0);
 }
 
-bool DebugPassFile::runOnModule(llvm::Module &module) {
-    run(module, false);
-    return true;
-}
-
-bool DebugPassFile::doInitialization(llvm::Module &module) {
-    run(module, true);
-    return true;
-}
-
-llvm::Pass *CreateDebugPassFile(int number, llvm::StringRef name, std::string dir) {
-    return new DebugPassFile(number, name, dir);
+llvm::PreservedAnalyses DebugPassFile::run(llvm::Module &M, llvm::ModuleAnalysisManager &MAM) {
+    run(M, false);
```

Comment:
Now `run` is invoked only with `false` as second parameter now, so the parameter needs to be removed.

Reply:
DebugPass was removed.